alfresco-health-processor-platform/src/main/java/eu/xenit/alfresco/processor/model/PropertyConverter.java

```diff
@@ -0,0 +1,23 @@
+package eu.xenit.alfresco.processor.model;
+
+import eu.xenit.alfresco.processor.service.ProcessorService;
+import eu.xenit.alfresco.processor.service.ProcessorServiceImpl;
+
+import java.security.InvalidParameterException;
+
+public interface PropertyConverter<T> {
```

Comment:
Again, looks like unnecessary overhead to me.

Example: `PropertyConverter.BOOLEAN` is not shorter than `Boolean::parseBoolean` and even worse: requires an additional step / link to follow to get a grasp of what is actually going on.